js/src/3rdparty/etherscan/links.js

```diff
@@ -15,15 +15,18 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 // NOTE: Keep 'isTestnet' for backwards library compatibility
-export const url = (isTestnet = false, netVersion = '0') => {
-  let prefix = '';
+export const url = (isTestnet = false, netVersion = '0', defaultPrefix = '') => {
+  let prefix = defaultPrefix;
 
   switch (netVersion) {
     case '2':
```

Comment:
we should remove `'2'` completely from this list, I don't know of any working morden explorer and etherscan recently told me they do not plan to maintain one.